contracts/burner/src/contract.rs

```diff
@@ -39,19 +38,18 @@ pub fn migrate(deps: DepsMut, env: Env, msg: MigrateMsg) -> StdResult<Response>
 
     let data_msg = format!("burnt {} keys", count).into_bytes();
 
-    Ok(Response {
-        messages: vec![SubMsg::new(send)],
-        attributes: vec![attr("action", "burn"), attr("payout", msg.payout)],
-        events: vec![],
-        data: Some(data_msg.into()),
-    })
+    Ok(Response::new()
+        .add_message(send)
+        .add_attribute(("action", "burn"))
```

Comment:
(I haven't looked at the std changes yet, starting with usage)

Note that we may well want `add_attributes` that takes a Vec, as there are cases in which I compose these attributes in my business logic and want to set them all, rather than incur the overhead of feeding them in one-by-one. Maybe these can be changed by adding those directly to the Response object rather than an intermediate Vec, but I think it could be useful. (Not here, but in some cosmwasm-plus contracts)